api/provider/aws/deploy_create.go

```diff
@@ -0,0 +1,133 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/errors"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (d *DeployProvider) Create(req models.CreateDeployRequest) (*models.Deploy, error) {
+	deployID := generateEntityID(req.DeployName)
+	fqDeployID := addLayer0Prefix(d.Config.Instance(), deployID)
+	familyName := fqDeployID
+
+	taskDefinition, err := d.renderTaskDefinition(req.DeployFile, familyName)
+	if err != nil {
+		return nil, err
+	}
+
+	taskDefinitionOutput, err := d.createTaskDefinition(taskDefinition)
+	if err != nil {
+		return nil, err
+	}
+
+	bytes, err := json.Marshal(taskDefinitionOutput)
```

Comment:
We don't need to do this part since it already exists in `req.DockerFile`

Reply:
Hmm, I would still rather marshal the output of `createTaskDefinition` and return that as part of `Create()` rather than passing the `req` back.